src/components/AlertsNotifications/AlertNotifications.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { View, Picker } from 'react-native';
+
+import { Gradient } from '../../shared/';
+import { TopButtons } from './TopButtons';
+import List from './ListAlertsAndNotifications';
```

Comment:
Ne znam koliko je dobra praksa da naziv importa bude drugaciji od naziva fajla koji se importuje? To bi mogli isto da prodiskutujemo...